### PR TITLE
CI Update - .NET Core 3.0 preview 9

### PR DESCRIPTION
**Pull request by mairaw**

### CODE REVIEW

File: xml/System.Net.Sockets/ProtocolFamily.xml [hunk 1]
```diff
@@ -919,11 +893,11 @@
     </Member>
     <Member MemberName="Packet">
       <MemberSignature Language="C#" Value="Packet" />
-      <MemberSignature Language="ILAsm" Value=".field public static literal valuetype System.Net.Sockets.ProtocolFamily Packet = int32(31)" />
+      <MemberSignature Language="ILAsm" Value=".field public static literal valuetype System.Net.Sockets.ProtocolFamily Packet = int32(65536)" />
       <MemberSignature Language="DocId" Value="F:System.Net.Sockets.ProtocolFamily.Packet" />
       <MemberSignature Language="VB.NET" Value="Packet" />
       <MemberSignature Language="C++ CLI" Value="Packet" />
-      <MemberSignature Language="F#" Value="Packet = 31" Usage="System.Net.Sockets.ProtocolFamily.Packet" />
+      <MemberSignature Language="F#" Value="Packet = 65536" Usage="System.Net.Sockets.ProtocolFamily.Packet" />
```
Comment (rpetrusha) [inline xml/System.Net.Sockets/ProtocolFamily.xml]: This is a suspicious change, but I verified it in the source. That's true of the other changes to underlying enum values as well -- all are enum members introduced in 3.0.

File: xml/System.Text.Encodings.Web/TextEncoder.xml [hunk 2]
```diff
@@ -199,6 +199,38 @@
           <paramref name="characterCount" /> is out of range.</exception>
       </Docs>
     </Member>
+    <Member MemberName="Encode">
```
Comment (rpetrusha) [inline xml/System.Text.Encodings.Web/TextEncoder.xml]: A new overload to document.

File: xml/System.Text.Json/JsonSerializer.xml [hunk 3]
```diff
@@ -373,12 +374,12 @@ There is remaining data in the stream.</exception>
...
       <Parameters>
         <Parameter Name="value" Type="System.Object" />
-        <Parameter Name="type" Type="System.Type" />
+        <Parameter Name="inputType" Type="System.Type" />
         <Parameter Name="options" Type="System.Text.Json.JsonSerializerOptions" />
       </Parameters>
       <Docs>
         <param name="value">The value to convert.</param>
-        <param name="type">The type of the <paramref name="value" /> to convert.</param>
+        <param name="inputType">To be added.</param>
```
Comment (rpetrusha) [inline xml/System.Text.Json/JsonSerializer.xml]:
```suggestion
        <param name="inputType">The type of the <paramref name="value" /> to convert.</param>
```

File: xml/System.Text.Json/JsonSerializer.xml [hunk 4]
```diff
@@ -409,13 +411,13 @@ Using a <xref:System.String> is not as efficient as using UTF-8 encoding since t
...
         <Parameter Name="value" Type="System.Object" />
-        <Parameter Name="type" Type="System.Type" />
+        <Parameter Name="inputType" Type="System.Type" />
         <Parameter Name="options" Type="System.Text.Json.JsonSerializerOptions" />
       </Parameters>
       <Docs>
         <param name="writer">The JSON writer.</param>
         <param name="value">The value to convert and write.</param>
-        <param name="type">The type of the <paramref name="value" /> to convert.</param>
+        <param name="inputType">To be added.</param>
```
Comment (rpetrusha) [inline xml/System.Text.Json/JsonSerializer.xml]:
```suggestion
        <param name="inputType">The type of the <paramref name="value" /> to convert.</param>
```

File: xml/System.Text.Json/JsonSerializer.xml [hunk 5]
```diff
@@ -506,14 +508,14 @@ Using a <xref:System.String> is not as efficient as using UTF-8 encoding since t
...
-        <Parameter Name="type" Type="System.Type" />
+        <Parameter Name="inputType" Type="System.Type" />
         <Parameter Name="options" Type="System.Text.Json.JsonSerializerOptions" />
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" />
       </Parameters>
       <Docs>
         <param name="utf8Json">The UTF-8 stream to write to.</param>
         <param name="value">The value to convert.</param>
-        <param name="type">The type of the <paramref name="value" /> to convert.</param>
+        <param name="inputType">To be added.</param>
```
Comment (rpetrusha) [inline xml/System.Text.Json/JsonSerializer.xml]:
```suggestion
        <param name="inputType">The type of the <paramref name="value" /> to convert.</param>
```

File: xml/System.Text.Json/JsonSerializer.xml [hunk 6]
```diff
@@ -569,12 +572,12 @@ Using a <xref:System.String> is not as efficient as using UTF-8 encoding since t
...
       <Parameters>
         <Parameter Name="value" Type="System.Object" />
-        <Parameter Name="type" Type="System.Type" />
+        <Parameter Name="inputType" Type="System.Type" />
         <Parameter Name="options" Type="System.Text.Json.JsonSerializerOptions" />
       </Parameters>
       <Docs>
         <param name="value">The value to convert.</param>
-        <param name="type">The type of the <paramref name="value" /> to convert.</param>
+        <param name="inputType">To be added.</param>
```
Comment (rpetrusha) [inline xml/System.Text.Json/JsonSerializer.xml]:
```suggestion
        <param name="inputType">The type of the <paramref name="value" /> to convert.</param>
```

File: xml/System.Windows.Documents/PresentationUIStyleResources.xml [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+  <AssemblyInfo>
+    <AssemblyName>PresentationUI</AssemblyName>
+    <AssemblyVersion>4.0.0.0</AssemblyVersion>
+  </AssemblyInfo>
+  <Base>
+    <BaseTypeName>System.Object</BaseTypeName>
+  </Base>
+  <Interfaces />
+  <Docs>
+    <summary>To be added.</summary>
```
Comment (rpetrusha) [inline xml/System.Windows.Documents/PresentationUIStyleResources.xml]:
```suggestion
    <summary>This class is used to create <see cref="T:System.Windows.ComponentResourceKey" /> objects, which require a public class.</summary>
```

File: xml/System.Windows.Forms/DataGridViewComboBoxEditingControl.xml [hunk 8]
```diff
@@ -429,6 +429,30 @@
...
+        <AssemblyVersion>4.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Void</ReturnType>
+      </ReturnValue>
+      <Parameters>
+        <Parameter Name="e" Type="System.EventArgs" Index="0" FrameworkAlternate="netcore-3.0" />
+      </Parameters>
+      <Docs>
+        <param name="e">To be added.</param>
```
Comment (rpetrusha) [inline xml/System.Windows.Forms/DataGridViewComboBoxEditingControl.xml]:
```suggestion
        <param name="e">An object that contains the event data.</param>
```

File: xml/System.Windows.Forms/DataGridViewComboBoxEditingControl.xml [hunk 9]
```diff
@@ -429,6 +429,30 @@
...
+        <summary>To be added.</summary>
```
Comment (rpetrusha) [inline xml/System.Windows.Forms/DataGridViewComboBoxEditingControl.xml]:
```suggestion
        <summary>Raises the <see cref="E:System.Windows.Forms.Control.HandleCreated" /> event.</summary>
```

File: xml/System.Windows.Forms/DataGridViewTextBoxEditingControl.xml [hunk 10]
```diff
@@ -434,6 +434,30 @@
...
+        <AssemblyVersion>4.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Void</ReturnType>
+      </ReturnValue>
+      <Parameters>
+        <Parameter Name="e" Type="System.EventArgs" Index="0" FrameworkAlternate="netcore-3.0" />
+      </Parameters>
+      <Docs>
+        <param name="e">To be added.</param>
```
Comment (rpetrusha) [inline xml/System.Windows.Forms/DataGridViewTextBoxEditingControl.xml]:
```suggestion
        <param name="e">An object that contains the event data.</param>
```

File: xml/System.Windows.Forms/DataGridViewTextBoxEditingControl.xml [hunk 11]
```diff
@@ -434,6 +434,30 @@
...
+        <summary>To be added.</summary>
```
Comment (rpetrusha) [inline xml/System.Windows.Forms/DataGridViewTextBoxEditingControl.xml]:
```suggestion
        <summary>Raises the <see cref="E:System.Windows.Forms.Control.HandleCreated" /> event.</summary>
```

**Comment by mairaw** [comment]
Good catch on the param name changes @rpetrusha! Thanks!

**Pull request merged**